.appveyor.yml

```diff
@@ -7,45 +7,127 @@ version: '{build}-{branch}'
-  - cmd: _build\installers\stackless-setup.msi /quiet /passive /qn /norestart
+      $WXInstaller = $BuildFolder + "wxPython3.0-win32-3.0.2.0-py27.exe"
+      If ((Test-Path $WXInstaller) -eq $False) {
+        " "
+        "--------- Installing  wxPython 3.0.2.0 ---------"
+        " "
+        $WXURL = "http://downloads.sourceforge.net/wxpython/wxPython3.0-win32-3.0.2.0-py27.exe"
+        Start-FileDownload $WXURL -Timeout 60000 -FileName $WXInstaller
+        & $WXInstaller "/VERYSILENT" "/SUPPRESSMSGBOXES"
+      }
 
-  - cmd: pip install commonmark==0.7.2
-  - cmd: pip install future==0.16.0
-  - cmd: pip install pillow==3.4.2
-  - cmd: pip install py2exe_py2==0.6.9
-  - cmd: pip install sphinx==1.4.8
-  - cmd: pip install https://eventghost.github.io/dist/dependencies/ctypeslib-0.5.6-cp27-none-any.whl
-  - cmd: pip install https://eventghost.github.io/dist/dependencies/pycrypto-2.6.1-cp27-none-win32.whl
-  - cmd: pip install https://eventghost.github.io/dist/dependencies/pywin32-220-cp27-none-win32.whl
-  - cmd: pip install https://eventghost.github.io/dist/dependencies/wxPython-3.0.2.0-cp27-none-win32.whl
+      $PyWinInstaller = $BuildFolder + "pywin32-220.win32-py2.7.exe"
+      If ((Test-Path $PyWinInstaller) -eq $False) {
+        " "
+        "-------------- Installing pywin32 --------------"
+        " "
+        $PyWinWheel = $BuildFolder + "pywin32-220-cp27-none-win32.whl"
+        $PywinPostInstall = $Env:PYTHON + "\Scripts\pywin32_postinstall.py"
+        $PyWinURL = "https://sourceforge.net/projects/pywin32/files/pywin32/Build%20220/pywin32-220.win32-py2.7.exe/download"
+        Start-FileDownload $PyWinURL -Timeout 60000 -FileName $PyWinInstaller
+        cd $BuildFolder
+        & "pip" "install" "wheel"
+        $ErrorActionPreference = "SilentlyContinue"
+        & "wheel" "convert" $PyWinInstaller
+        $ErrorActionPreference = "Continue"
+        & "pip" "install" $PyWinWheel
+        & "python" $PywinPostInstall "-install" "-silent" "-quiet"
+        cd ..\..
+      }
 
-  # jinja2 >= 2.9 has an async module which py2exe has problems with (build of eg will fail)
-  - cmd: pip install jinja2==2.8.1
+      " "
+      "-------------- Installing Modules --------------"
+      " "
+      & "pip" "install" "--upgrade" "setuptools"
+      & "pip" "install" "commonmark==0.7.2"
 
-  # comtypes==1.1.3
-  #   Could not find a version that satisfies the requirement comtypes==1.1.3 (from versions: 0.6.4, 1.0.0, 1.1.0, 1.1.1, 1.1.3.post2)
-  #   No matching distribution found for comtypes==1.1.3
-  - cmd: pip install https://github.com/enthought/comtypes/archive/1.1.3.zip
+      $Jinja = $SitePackages + "jinja2"
+      If ((Test-Path $Jinja) -eq $False) {
+        & "pip" "install" "jinja2==2.8.1"
+      }
+
+      $CTypesLib = $SitePackages + "ctypeslib"
+      If ((Test-Path $CTypesLib) -eq $False) {
+        & "pip" "install" "-e" "svn+http://svn.python.org/projects/ctypes/trunk/ctypeslib/#egg=ctypeslib-dev"
```

Comment:
I am not sure as to why it is not grabbing the cache data and creating it.. it does on my appveyor.. perhaps it's because it's a PR??